v5_MIGRATION_GUIDE.md

```diff
@@ -212,3 +213,73 @@ class MyActivity : FragmentActivity() {
 }
 ```
 
+## Local Payment
+
+The Local Payment integration has been updated to allow for more flexibility with browser 
+switching and result handling.
+
+`LocalPaymentLauncher` has been added to handle the browser switching portion of the local payment 
+flow for payment authorization via bank login. This class is used to handle browser switch
+results upon returning to your app from the web flow, therefore must be instantiated in or
+before the `OnResume` method of your Activity or Fragment.
+
+`BraintreeClient` and `LocalPaymentClient` no longer require references to Fragment or Activity and
+do not need to be instantiated in `OnCreate`.
+
+```diff
+class MyActivity : FragmentActivity() {
+
++   private lateinit var localPaymentLauncher: localPaymentLauncher
+    private lateinit var braintreeClient: BraintreeClient
+    private lateinit var localPaymentClient: LocalPaymentClient
+
+    @override fun onCreate(savedInstanceState: Bundle?) {
++       // can initialize clients outside of onCreate if desired
+-       initializeClients()
++       localPaymentLauncher = LocalPaymentLauncher() { localPaymentBrowserSwitchResult ->
++           localPaymentClient.onBrowserSwitchResult(localPaymentBrowserSwitchResult) { 
++               localPaymentNonce, error ->
++                   // handle local payment nonce or error
++           }
++       }
+    }
+
+    // ONLY REQUIRED IF YOUR ACTIVITY LAUNCH MODE IS SINGLE_TOP
+    @override fun onNewIntent(intent: Intent) {
+        setIntent(intent)
```

Comment:
Once the refactor's complete we should be able to remove `setIntent(intent)` 🙏 😮‍💨